src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,96 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.26.0.dev0 (02/11/2020)
+------------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Revert "Switch to dot idea project type (#8745)" (#9096)
+  `PR #9096 <https://github.com/pantsbuild/pants/pull/9096>`_
```

Comment:
@wisechengyi is this correct?

Reply:
Would be bug fixes I think. 

When the revert of revert happens, it will be `Refactoring, Improvements, and Tooling` for having almost no changes from user point of view.

Thank you!